tests/rdf_tests/test_rdf_basic.py

```diff
@@ -0,0 +1,102 @@
+from os.path import abspath, dirname
+import sys
+from pathlib import Path
+
+project_dir = Path(dirname(dirname(dirname(abspath(__file__)))))
+sys.path.insert(0, str(project_dir))
+tests_dir = project_dir / "tests"
+sys.path.insert(0, str(tests_dir))
+DATA_DIR = project_dir / "tests" / "rdf_tests" / "dat"
+
+import kglab
+from icecream import ic
+
+from rdf_tests.rdflib_tools import read_manifest
+
+
+def parse_manifest(dir_, file_):
+    """ Retrieve tests manifest """
+    manifest_file = DATA_DIR / dir_ / file_
+    ic(manifest_file)
+
+    yield read_manifest(manifest_file)
+
+def clean_filepath(filepath):
+    """ Clean paths loaded from RDF manifest """
+    return str(Path(str(filepath))).replace("file:", "")
+    
+
+def run_test(t):
```

Comment:
Beware, you support here only `mf:QueryEvaluationTest` and not the other kind of tests (updates, syntax...).